examples/ui.apps/src/main/content/jcr_root/apps/forms-components-examples/components/form/tabsonleft/.content.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+  jcr:description="Adaptive Form Vertical Tab Container component, used to group other components and apply a common style or layout."
+  jcr:primaryType="cq:Component"
+  jcr:title="Adaptive Form Vertical Tabs"
+  sling:resourceSuperType="core/fd/components/form/tabsonleft/v1/tabsonleft"
```

Comment:
I'd not prefer having left anywhere in the folder name in the resource type (because of RTL scenarios).
lets use verticaltabs